docs/examples-c-wasi.md

```diff
@@ -10,7 +10,7 @@ This example shows off how to instantiate a wasm module using WASI imports.
 ## Wasm Source code
 
 ```rust,ignore
-{{#include ../examples/wasi/wasm/wasi.c}}
+{{#include ../examples/wasi/wasm/wasi.rs}}
```

Comment:
This is `examples-c-wasi.md` so if we don't have a C example yet, we should just say so :-).

Reply:
Oh in this case this is actually the source code for the wasm module, the C code to run it is pasted below